templates/config.yml

```diff
@@ -93,6 +93,9 @@ blockchain_adapters:
 # You can also set a testnet insight url to use:
 #   insight_test_url: "https://test.localhost/api"
 
+# Websocket Insight Url
+insight_websocket_url: "wss://insight.mycelium.com"
```

Comment:
Abuse Bitpay server by default it's too not a great idea I think :)
Better I put `localhost` there and comment about it. Users will be decide which server use.